### ISSUE TITLE
Can denoiser be turned off?

### ISSUE THREAD

**Issue by Rainboom-Dash** [xinntao/Real-ESRGAN#924]
I want to be able to just completely disable denoising for the x4plus-anime model
It's destroying images, some are clean PNGs that don't need it in the first place
Is this possible?
An option to reduce would be nice, too. I'm finding better results running highly noisy images through waifu2x first, because if it detects a lot of noise in an image, it turns the denoiser really aggressive and blurs everything

**Comment by limitlesslab** [comment]
You would need to retrain the model yourself with modified training settings. The authors of this arch have already moved on and are unlikely to release anything new. I can help retrain if you can't do it yourself, but it won't be free. You can find me on discord, my username is 'enjoynewlife'

**Comment by Rainboom-Dash** [comment]
I was hoping it could just easily be switched off
I know strength adjustment would require re-training
Oh well, thanks

**Comment by limitlesslab** [comment]
Just know that these officially released models are extremely undertrained and are capable of so much more.

**Comment by Rainboom-Dash** [comment]
I actually think the anime 6b model is pretty good, main issue is very strong denoiser
I found another model that also is good but has a less strong denoiser
I do agree the general model is bad, though

**Comment by Rainboom-Dash** [comment]
Yeah, guess it's part of the model, oh well
I found alternatives for both anime and real life images (I'll still occasionally use x4plus-anime)
I doubt the general image upscaler would work well for anime, and there's _significantly_ better real life image upscalers, at this point

**Comment by Rainboom-Dash** [comment]
sudo RealESRGAN2x Dropout for anime
4xRealWebPhoto_v3_atd for real life images
4xRealWebPhoto_v4_dat2 is also good, a little different

**Comment by ukr-projects** [comment]
thanks
